tests/e2e/specs/orderList_spec.js

```diff
@@ -39,13 +39,14 @@ describe('my orders', () => {
 
   beforeEach(() => {
     cy.visit('/');
+    cy.createCustomer(customer);
     cy.login(customer);
   });
 
   it('shows my orders', () => {
     cy.createOrder(cartDraft1, orderDraft1);
     cy.createOrder(cartDraft2, orderDraft2);
-    cy.get('[data-test=my-orders-button]', { timeout: 20000 }).click();
+    cy.get('[data-test=my-orders-button]', { timeout: Cypress.config('graphqlTimeout') }).click();
     cy.get('[data-test=order-list]')
       .should('have.length', 2)
```

Comment:
And instead, I would put it here, and in the `get`. In both places, because the 1st one will work when there's 1 order and the second one when there are 2.